client/charts/js/components/FilterSet.jsx

```diff
@@ -241,6 +241,9 @@ export default function FilterSet({ filters, handleFilterChange, filterParameter
 	const components = filters.map((filter, index) => {
 		if (filter.name === 'search' || filter.name === 'tags') {
 			return
+		} else if (!filterParameters.hasOwnProperty(filter.name)) {
+			console.warn(`no filter parameters defined for filter "${filter.name}"`)
+			return
```

Comment:
Part of me wonders if we should use `invariant` here, but that wouldn't have fixed our problem since `invariant` disables logging in prod.